www/utils/swrCache.ts

```diff
@@ -0,0 +1,12 @@
+import type { Cache } from 'swr';
+
+const memoryCache = new Map<string, any>();
```

Comment:
Looks like this is just an in memory cache not actually using indexedDB

Reply:
Reverted to localStorage